protocol/oracle-manager/storage.dump.sol

```diff
@@ -64,18 +61,15 @@ library NodeDefinition {
         REDUCER,
         EXTERNAL,
         CHAINLINK,
-        PYTH,
-        PriceDeviationCircuitBreaker,
-        UNISWAP,
-        StalenessFallbackReducer
+        PYTH
     }
     struct Data {
         bytes32[] parents;
         NodeType nodeType;
         bytes parameters;
     }
     function load(bytes32 id) internal pure returns (Data storage data) {
-        bytes32 s = keccak256(abi.encode("io.synthetix.oracle-manager.Node", id));
+        bytes32 s = keccak256(abi.encode("Node", id));
```

Comment:
might want to be careful with this name here, its so generic that it could easily clash with another module thats added at a later date that may happen to have the same name. maybe going further to name this `OracleManagerNode` would be safer?